### DISCUSSION TITLE
使用 Android Pad 进行（前端）开发 · TurboKre的博客 | Developing (Frontend) on an Android Pad · TurboKre's Blog

### DISCUSSION THREAD

**Discussion by giscus** [ohmykreee/kreee-blog#13] [Comments]
# 使用 Android Pad 进行（前端）开发

使用 Android Pad 进行（前端）开发 - A Kreee's Blog, powered by GitHub Pages, Hugo and Bilberry Premium Theme for Hugo.

https://blog.kre3.net/article/frontend-dev-with-android-pad/

**Comment by shenmo7192** [comment]
这里推荐一下atilo，操作会比这个更简单些哦
而且atilo-cn会自动选择国内源

---
目前所有基于容器的termux Linux方案的io都有很大问题，尽量能在termux原生运行的都在termux原生，本人亲测的是同样的nginx+php+kodexplorer，在容器里加载网页大概需要15秒，termux原生只用1.1秒

**Reply by shenmo7192** [reply]
https://github.com/YadominJinta/atilo

**Reply by ohmykreee** [reply]
刚刚看了一下 atilo，感觉它的底层实现 chroot 和 proot 差不多啊（不知道对不对）。

感觉我现在使用的最大瓶颈还不是 IO，反而是内存。我的米板买的是 6Gb 内存规格的，MIUI 也没有刷开发版或者 ROOT （~~主要是懒~~），也不知道系统的真实内存占用。我在 VSCode 里装了一个性能监测插件，发现 Termux 被分配了 5.5Gb 左右的内存，开 code-server 空载占用已经到达了 4.8 Gb 左右。这就导致了经常写着写着 Termux 的后台就被 MIUI 给杀掉了，即使在内存清理里给了白名单。

不过我本来也没有打算在 Pad 上做主力开发，只是 ~~闲着没事干~~ 探索一种新的可能。（以及最近疯狂沉迷 osu!mania，导致我年前挖的坑直接就被弃了，我的账户主页可以在[我的个人主页](https://www.ohmykreee.top)里找到。）

**Reply by shenmo7192** [reply]
目前termux上所有的免root的Linux方案底层都是一样的，我自己在用atilo所以就推荐了，因为atilo-cn都自动用国内mirror

如果是内存不足的话那从termux原生启动的区别也不大，不过个人还是建议原生，因为磁盘性能提升真的很明显

我之前为了玩osu!还买了个数位板 :joy 然而最后还是触屏真香了
